docs/source/reference/ha.rst

```diff
@@ -0,0 +1,202 @@
+High availability deployment
+============================
+
+|st2| has been systematically built with High availability(HA) as a goal. The exact deployment
+steps to achieve HA depend on the specifics of the infrastructure in which |st2| is deployed. This
+guide covers a brief explanation on various |st2| services, how they interact and the external
+services necessary for |st2| to function.
+
+In the section :doc:`/install/overview` a detailed picture and explanation on how single-box deployments
+work is provided. Repeating the picture here to keep context and use as a reference to layer on some
+HA deployment specific details.
+
+.. figure :: /_static/images/st2-deployment-big-picture.png
```

Comment:
I went ahead and added a big picture. Its is not as nice and detailed as the picture from overview. Hopefully what I have created can work as a start that can be built-on and is yet acceptable in the short-term.